test/integration/targets/docker_config/tasks/main.yml

```diff
@@ -0,0 +1,2 @@
+- include_tasks: test_docker_config.yml
+  when: ansible_os_family != 'RedHat' or ansible_distribution_major_version != '6'
```

Comment:
If the tests have a minimum version requirement for docker, check for that instead of distribution version.